games-util/gamemode/gamemode-1.4.ebuild

```diff
@@ -0,0 +1,117 @@
+	GAMEMODE_GIT_PTR="master"
+	inherit git-r3
+else
+	GAMEMODE_GIT_PTR="${PV}"
+	SRC_URI="https://github.com/FeralInteractive/gamemode/releases/download/${GAMEMODE_GIT_PTR}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+
+RDEPEND="
+	>=sys-apps/systemd-236[${MULTILIB_USEDEP}]
+	sys-auth/polkit
+	sys-libs/libcap
+"
+DEPEND="${RDEPEND}"
+
+DOCS="CHANGELOG.md LICENSE.txt README.md example/gamemode.ini"
+
+pkg_pretend() {
+	elog
+	elog "GameMode needs a kernel capable of SCHED_ISO to use its soft realtime"
+	elog "feature. Examples of kernels providing that are sys-kernel/ck-source"
+	elog "and sys-kernel/pf-sources."
+	elog
+	elog "Support for soft realtime is completely optional. It may provide the"
+	elog "following benefits with systems having at least four CPU cores:"
+	elog
+	elog "  * more CPU shares allocated exclusively to the game"
+	elog "  * reduced input lag and reduced thread latency"
+	elog "  * more consistent frame times resulting in less microstutters"
+	elog
+	elog "You probably won't benefit from soft realtime mode and thus don't need"
+	elog "SCHED_ISO if:"
+	elog
+	elog "  * Your CPU has less than four cores because the game may experience"
+	elog "    priority inversion with the graphics driver (thus heuristics"
+	elog "    automatically disable SCHED_ISO usage then)"
+	elog "  * Your game uses busy-loops to interface with the graphics driver"
+	elog "    but you may still force SCHED_ISO per configuation file, YMMV,"
+	elog "    it depends on the graphics driver implementation, i.e. usage of"
+	elog "    __GL_THREADED_OPTIMIZATIONS or similar."
+	elog "  * If your game causes more than 70% CPU usage across all cores,"
+	elog "    SCHED_ISO automatically turns off and on depending on usage and"
+	elog "    is processed with higher-than-normal priority then (renice)."
+	elog "    This auto-switching may result in a lesser game experience."
+	elog
+	elog "GameMode has optional support for adjusting nice and ioprio of games"
+	elog "running with it. You may need to adjust your PAM limits to make use"
+	elog "of this. You can run the following command to test your settings:"
+	elog
+	elog "# gamemoded -t"
+	elog
+	elog "GameMode supports GPU optimizations. It defaults to OFF. Any"
+	elog "damage resulting from usage of this is your own responsibility."
+	elog
+	elog "For more info look at:"
+	elog "https://github.com/FeralInteractive/gamemode/blob/${GAMEMODE_GIT_PTR}/README.md"
+	elog
+}
+
+multilib_src_configure() {
+	local myconf=()
+	if ! multilib_is_native_abi; then
+		myconf+=(
+			-Dwith-examples=false
+			-Dwith-daemon=false
+		)
+	fi
+	meson_src_configure "${myconf[@]}"
+}
+
+multilib_src_compile() {
+	eninja
+}
+
+multilib_src_install() {
+	DESTDIR="${D}" eninja install
+	if multilib_is_native_abi; then
+		insinto /etc/security/limits.d
+		newins - 45-gamemode.conf <<-EOF
+			@gamemode - nice -10
+		EOF
+	fi
+}
+
+pkg_postinst() {
+	enewgroup gamemode
```

Comment:
This needs the new acct-group method of handling groups :\

These comments apply to -9999 as well.

Reply:
Does it need EAPI=7 also in that case?

Reply:
It shouldn't, but I haven't tested and verified that.